lightningd/channel.h

```diff
@@ -125,7 +125,7 @@ static inline u16 to_self_delay(const struct channel *channel, enum side side)
  * @funding_txout: The commitment transaction output number.
  * @funding_satoshis: The commitment transaction amount.
  * @local_msatoshi: The amount for the local side (remainder goes to remote)
- * @feerate_per_kw: feerate per kiloweight (satoshis)
+ * @feerate_per_kw: feerate per kiloweight (satoshis) for the settlement transaction
```

Comment:
This is actually not a term used in the spec.  It's the "commitment transaction", but it's also used in the HTLC transactions.